samples/HostedConsoleApp/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Hosting;
+using System.CommandLine.Invocation;
+using System.Reflection;
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace HostedConsoleApp
+{
+    public static class Program
+    {
+        public static Task Main(string[] args) => new CommandLineBuilder(
+            new RootCommand
+            {
+                Handler = CommandHandler.Create<IHost, string>(Run),
+                TreatUnmatchedTokensAsErrors = false
```

Comment:
We shouldn't need this anymore, correct?

Reply:
Remoed